test/Microsoft.AspNet.JsonPatch.Test/NestedObjectTests.cs

```diff
@@ -1047,6 +1144,29 @@ public void ReplaceInListInvalidInvalidPositionTooLargeWithSerialization()
         }
 
         [Fact]
+        public void ReplaceInListInvalidInvalidPositionTooLarge_LogError()
```

Comment:
`ReplaceInListInvalid_PositionTooLarge_LogError` would it make more sense?
nit: Also in general  would changing the suffix of the tests form `LogError` to `LogsError` make sense?